marcus      2006-11-11 02:56:18 UTC

  FreeBSD ports repository

  Modified files:
    net/avahi            Makefile distinfo 
  Added files:
    net/avahi/files      patch-avahi-daemon_dbus-protocol.c 
  Log:
  Update to 0.6.15, and fix the build with D-BUS 1.0.
  
  Revision  Changes    Path
  1.43      +3 -3      ports/net/avahi/Makefile
  1.12      +3 -3      ports/net/avahi/distinfo
  1.3       +11 -0     ports/net/avahi/files/patch-avahi-daemon_dbus-protocol.c (new)

--- avahi-daemon/dbus-protocol.c.orig	Fri Nov 10 21:50:07 2006
+++ avahi-daemon/dbus-protocol.c	Fri Nov 10 21:50:55 2006
@@ -1067,7 +1067,7 @@ static int dbus_connect(void) {
     if (dbus_bus_request_name(
             server->bus,
             AVAHI_DBUS_NAME,
-#if (DBUS_VERSION_MAJOR == 0) && (DBUS_VERSION_MINOR >= 60)
+#if ((DBUS_VERSION_MAJOR == 0) && (DBUS_VERSION_MINOR >= 60)) || DBUS_VERSION_MAJOR > 0
             DBUS_NAME_FLAG_DO_NOT_QUEUE,
 #else
             DBUS_NAME_FLAG_PROHIBIT_REPLACEMENT,

